File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        l, r = 0, len(numbers) - 1
        left, right = 0, 0
        while l < r:
            if numbers[l] + numbers[r] > target:
                r -= 1
            elif numbers[l] + numbers[r] < target:
                l += 1
            else:
                left = l + 1
                right = r + 1
                break
        return [left, right]

File: Data Structures & Algorithms/merge-sorted-array/submission-1.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int right = n + m - 1;
        int left = n - 1;
        int mid = m - 1;


        while (left >= 0) {
            if (mid >= 0 && nums1[mid] > nums2[left]) { 
                nums1[right] = nums1[mid];
                right--;
                mid--;
            } else {
                nums1[right] = nums2[left];
                right--;
                left--;
            }
        }

    }
}